alibi_detect/cd/tensorflow/mmd.py

```diff
@@ -139,7 +139,7 @@ def get_config(self) -> dict:
         # kernel logic
         if isinstance(self.kernel, GaussianRBF):
             # If default kernel, we don't need to spec
-            sigma = self.kernel.sigma.numpy().tolist() if not self.infer_sigma else None
+            sigma = self.kernel.sigma.numpy() if not self.infer_sigma else None
```

Comment:
What's the reason for the change `list -> ndarray`?

Reply:
I previously had the `tolist()` as when the `get_config` output is used in the later `save_detector` stuff coming in part III, `sigma` needs to be a list so it can be saved to TOML. However, since its typed as ndarray in the pydantic model, I realised this meant the `get_config` output would actually raise a `ValidationError` when passed to `validate_config`. Think its important the `get_config` outputs always return valid configs, so will offload this conversion to list to `saving.py` in part III instead. Think this is in more inline with the general strategy anyway, of `get_config` returning a "resolved" config, and then `saving.py` being responsible for turning it into an "unresolved" config.